### ISSUE TITLE
Is k-diffusion included with this optimized version of SD?

### ISSUE THREAD

**Issue by JoshuaKimsey** [basujindal/stable-diffusion#54]
Apologizes if this is the wrong place to ask. Was just curious if K-Diffusion/k_lms was added for the 0.5 release. I've been using a different setup, a fork or companion of this version with Gradio, that uses it and quite like the results. 

If K-Diffusion isn't available yet, any chance it could be added as a sampling option? :)

**Comment by Orthestral** [comment]
I'd be very interested in using k_lms and k_euler_a as well without being limited to 512x512

**Comment by TheEnhas** [comment]
Yeah I'd like more samplers too, PLMS is okay but sometimes k_euler_ancestral and others are better depending on the image.

**Comment by JoshuaKimsey** [comment]
It would be awesome to see more samplers implemented here directly. One variant of this by [hlky](https://github.com/hlky/stable-diffusion), which is used by the [guide](https://rentry.org/guitard) the [Stable Diffusion Subreddit](https://www.reddit.com/r/StableDiffusion/wiki/guide) suggests, adds a large number of samplers, including k-lms, k_euler_a, and more. It might be worth looking into to add a number of different options for samplers in the base Optimized SD. It also adds the Gradio modifications to support that functionality too. 

I'm also not sure how much it is needed compared to the other one I linked to, I need to test out this version of SD fully on Linux, but I'd be happy to help add any additional support for Linux-based systems as well. :)

**Comment by throwaway-mezzo-mix** [comment]
+1 on this, I would love more samplers.
